src/components/embed/ui/molecules/copy-to-clipboard/CopyToClipboard.tsx

```diff
@@ -0,0 +1,92 @@
+import { useEffect, useState } from "react";
+import styled from "styled-components";
+import browser from "webextension-polyfill";
+import copy from "copy-to-clipboard";
+import { Check, Copy02 } from "@untitled-ui/icons-react";
+import { toast } from "react-toastify";
+import { Text } from "~components/embed/ui";
+
+interface CopyToClipboardProps {
+  text: string;
+  copySuccess?: string;
+  iconSize?: number;
+  label?: string;
+  labelStyle?: React.CSSProperties;
+  labelAs?: typeof Label;
+  showToast?: boolean;
+  onCopy?: (isCopied: boolean) => void;
+}
+
+const CopyButton = styled.button`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  cursor: pointer;
+  border: none;
+  background: none;
+  gap: 8px;
+`;
+
+const Label = styled(Text).attrs({
+  noMargin: true,
+})`
+  overflow: hidden;
+`;
+
+export function CopyToClipboard({
+  text,
+  copySuccess,
+  iconSize = 16,
+  label,
+  labelStyle,
+  showToast = true,
+  onCopy,
+  labelAs = Label,
```

Comment:
Only `text` is needed here. Even after this component is used in all places where we have a copyable feature, all instances should look the same, so there's no need to add plenty of customization options that we are not using, as this can easily lead to unintentional inconsistencies when instantiating this component in different places. Also, this makes this component harder to change/maintain to make sure all props work fine after we do any changes to it, even if those are not used.